packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -0,0 +1,474 @@
+	verticalId?: string;
+	onPreview: ( design: Design ) => void;
+}
+
+const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
+	locale,
+	onSelect,
+	onPreview,
+	onUpgrade,
+	designs,
+	premiumBadge,
+	categorization,
+	previewOnly = false,
+	hasDesignOptionHeader = true,
+	isPremiumThemeAvailable,
+	onCheckout = undefined,
+} ) => {
+	const hasCategories = !! categorization?.categories.length;
+	const filteredDesigns = useMemo( () => {
+		const result = categorization?.selection
+			? filterDesignsByCategory( designs, categorization.selection )
+			: designs.slice(); // cloning because otherwise .sort() would mutate the original prop
+
+		result.sort( sortDesigns );
+		return result;
+	}, [ designs, categorization?.selection ] );
+	return (
+		<div>
+			{ categorization && hasCategories && (
+				<>
+					<UnifiedDesignPickerCategoryFilter
+						categories={ categorization.categories }
+						onSelect={ categorization.onSelect }
+						selectedSlug={ categorization.selection }
+					/>
+				</>
+			) }
+			<div className={ 'design-picker__grid' }>
+				{ filteredDesigns.map( ( design ) => (
+					<DesignButtonContainer
+						key={ design.slug }
+						design={ design }
+						locale={ locale }
+						onSelect={ onSelect }
+						onPreview={ onPreview }
+						onUpgrade={ onUpgrade }
+						premiumBadge={ premiumBadge }
+						highRes={ false }
+						hideFullScreenPreview={ false }
+						hideDesignTitle={ false }
+						isPremiumThemeAvailable={ isPremiumThemeAvailable }
+						previewOnly={ previewOnly }
+						hasDesignOptionHeader={ hasDesignOptionHeader }
+						onCheckout={ onCheckout }
+					/>
+				) ) }
+			</div>
+		</div>
+	);
+};
+
+const GerneratedDesignPicker: React.FC< GeneratedDesignPickerProps > = ( {
+	locale,
+	designs,
+	verticalId,
+	onPreview,
+} ) => {
+	const isMobile = useViewportMatch( 'small', '<' );
+
+	return (
+		<div className="design-picker__grid">
+			{ designs.map( ( design, index ) => {
+				const previewUrl = getDesignPreviewUrl( design, {
+					language: locale,
+					verticalId,
+					viewport_width: isMobile ? MOBILE_VIEWPORT_WIDTH : DEFAULT_VIEWPORT_WIDTH,
+					viewport_height: DEFAULT_VIEWPORT_HEIGHT,
+					use_screenshot_overrides: true,
+				} );
+				return (
+					<div className="design-button-container" key={ `generated-design__${ index }` }>
+						<div className="design-picker__design-option">
+							<button className="generated-design-thumbnail" onClick={ () => onPreview( design ) }>
+								<span className="generated-design-thumbnail__image">
+									<ThemePreview
+										url={ previewUrl }
+										viewportWidth={ isMobile ? MOBILE_VIEWPORT_WIDTH : DEFAULT_VIEWPORT_WIDTH }
+									/>
+								</span>
+							</button>
+						</div>
+					</div>
+				);
+			} ) }
+		</div>
+	);
+};
+
+const UnifiedDesignPicker: React.FC< UnifiedDesignPickerProps > = ( {
+	locale,
+	onSelect,
+	onPreview,
+	onUpgrade,
+	verticalId,
+	staticDesigns = getAvailableDesigns( {
```

Comment:
I think the default value here is unused. We never show UDP in the Anchor FM flow, and I see that we always pass static designs to this UDP component. Can we remove it for simplicity?